JavaAssignments/src/saket/FtoCdegree.java

```diff
@@ -0,0 +1,22 @@
+package saket;
+
+public class FtoCdegree {
+	int Fahrenheit = 20;
+	int celcius;
+
+	void formula() {
+
+		celcius = (Fahrenheit - 32) * 5 / 9;
+		System.out.println("Conversion of Fahrenheit " + Fahrenheit + "into Celcius is" + celcius);
+
+	}
+
+	public static void main(String[] args) {
+
+		FtoCdegree A = new FtoCdegree();
```

Comment:
You should consider parameters in formula method.
Now consider, I want to convert 25 F to C... Though you have written method to convert that but i can't use your method as it is not taking any parameters. 

it should be ..

double formula(double fahrenheit)
{

}

NOTE: double is a datatype like float. you can use float also..